fe/be-java-extensions/max-compute-scanner/src/main/java/org/apache/doris/maxcompute/MaxComputeJniScanner.java

```diff
@@ -124,6 +129,7 @@ protected void initTableInfo(ColumnType[] requiredTypes, String[] requiredFields
         }
         // reorder columns
         List<Column> columnList = curTableScan.getSchema().getColumns();
+        columnList.addAll(curTableScan.getSchema().getPartitionColumns());
```

Comment:
Does `curTableScan.getSchema().getColumns()` contains partition columns?

Reply:
not contains